client/analytics/report/orders/index.js

```diff
@@ -13,11 +13,22 @@ import { ReportFilters } from '@woocommerce/components';
 /**
  * Internal dependencies
  */
-import { advancedFilters, filters } from './config';
-import OrdersReportChart from './chart';
+import { advancedFilters, charts, filters } from './config';
 import OrdersReportTable from './table';
+import ReportChart from 'analytics/components/report-chart';
+import ReportSummary from 'analytics/components/report-summary';
 
 export default class OrdersReport extends Component {
+	getSelectedChart() {
```

Comment:
After doing this to all reports, this function could go into `ReportChart` to further dry this up